sketcherMaeReading.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
```

Comment:
Does this file need to be added to the `install` list in CMakeLists.txt?
It looks like all the other header files are there, but I'm not sure all of them are actually needed (that'd be another PR to fix).

Reply:
I think that this is also private. We could expose it, though.

Reply:
Na, I think it makes sense to do a separate PR to only install the bits that are actually required to use the library. 

I was just pointing it out here to be consistent; definitely not required

Reply:
Na, I think it makes sense to do a separate PR to only install the bits that are actually required to use the library. 

I was just pointing it out here to be consistent; definitely not required